src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain methods directly to the result of `split` without assigning it to a variable first.